scripts/common.sh

```diff
@@ -104,7 +104,12 @@ _golint() {
 }
 
 _go_vet() {
-  go vet $(_test_dirs)
+  for d in $(_test_dirs)
```

Comment:
```suggestion
  for d in "$(_test_dirs)"
```

Reply:
done

Reply:
restored ... this one broke build

Reply:
hmm... wonder why? do you remember the error (just out of curiosity)